### PR TITLE
Header section

### PR DESCRIPTION
**Pull request by KebebewAbabu**
# Header Section Of the Mobile Version of my portfolio.
  # The header section branch contains code for the rest of the portfolio pages.
- [ ] Bug fix (unexpected generic font-family).
- [ ] New feature ( font-family; Poppins, Arial, Helvetica, sans-serif;)
  - [ ] Run local stylelint config command(npx stylelint "**/*.{css,scss}" --fix  after changing the font.
# Checklist
 [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

**Comment by KebebewAbabu** [comment]
# Header Section Of the Mobile Version of my portfolio.
  # The header section branch contains code for the rest of the portfolio pages which can be found in the images below.
- [ ] Bug fix (unexpected generic font-family).
  - [ ] Run local stylelint config command(npx stylelint "**/*.{css,scss}" --fix  after changing the font.
# Checklist
 [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

**Comment by KebebewAbabu** [comment]
Portfolio-project
-.github/workflows - linters.yml

- Package.json
- index.html
- /CSS
  - style.css
- /img
  -/icons
  - angellist.svg
  - circle 2 header.svg
  - down-arrow.svg
  - email.png
  - github.svg
  - hamburger.svg
  - linkedIn.svg
  - medium.svg
  - twitter.svg
  - vector.svg
  - work.svg
    shapes.png
- .stylelintrc.json
- .hintrc

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @KebebewAbabu,


Great job so far 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️

- [x] Please, kindly update the styling in your design and ensure the language and framework are aligned horizontally, and add the correct hamburger icon below:

*figma design*
span

*harmburger*
b






## Optional suggestions
**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+            <li class="list-item"><a href="#" class="list-links">Css</a></li>
+          </ul>
+          <ul class="lists biggi">
+            <li class="label-txt">Frameworks</li>
+            <li class="list-item"><a href="#" class="list-links">Bootstrap</a></li>
+            <li class="list-item"><a href="#" class="list-links">Ruby on Rails</a></li>
+            <li class="list-item"><a href="#" class="list-links">RSPec</a></li>
+            <li class="list-item"><a href="#" class="list-links">CapyBara</a></li>
+            <li class="list-item"><a href="#" class="list-links">Selenium</a></li>
+          </ul>
```
Comment (Ginohmk) [inline index.html:45]: - Please align the `Language framework and skill` session correctly as it is on the Figma design.
- [Optional] Please change the color of the social media icons to `background: #FFFFFF;`  as it on the Figma design,  You can try extracting the exact icon from the Figma template.


##### Your design 
Screenshot from 2022-06-30 16-57-35


##### Figma design
Screenshot from 2022-06-30 16-53-02

**Review by Ginohmk — CHANGES_REQUESTED** [review]
Hi @KebebewAbabu , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

keep-trying-youre-almost-there

 ## Highlights
- no linter errors :+1: 
- Good commit messages :100: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ugboaja-Uchechi — APPROVED** [review]
Hi @KebebewAbabu, 

## STATUS: APPROVED 🥳 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Your design is nice 👍 
- Linters are passing ✔️ ✔️ 
- Your repo looks professional 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**